### ISSUE TITLE
Bug: Melton City, AU. HTTPError 403: Client Error: Forbidden URL

### ISSUE THREAD

**Issue by UncleSam002** [mampfes/hacs_waste_collection_schedule#940]
Hi, thanks for such a helpful Addon! However I'm running into some minor problems.

HA Logs show a HTTPError 403: Client Error: Forbidden URL Error when executing `response.raise_for_status()`. Lines 35 - 42

in `/custom_components/waste_collection_schedule/waste_collection_schedule/source/melton_vic_gov_au.py`

```
        response = session.get("https://www.melton.vic.gov.au/My-Area")
        response.raise_for_status()

        response = session.get(
            "https://www.melton.vic.gov.au/api/v1/myarea/search",
            params={"keywords": self._street_address},
        )
        response.raise_for_status()
```

I have manually obtained my particular `geolocationid` and removed the above code and manually inserted my `geolocationid` to make it work for me but I cannot figure out how to properly set up authentication for the above URLs.

**Comment by 5ila5** [comment]
They filter traffic to detect and block strange requests or page scraping.
Adding some additional request at the beginning avoids this (Only tested on my machine) fixed by #942.
It might break again if they use more strict scraping protection or change blocking policies, but the quick fix should work for now.

**Comment by UncleSam002** [comment]
Thanks @5ila5. Can confirm it now works on my HA OS instance as well.
